packages/card/docs/index.mdx

```diff
@@ -1 +1,38 @@
+import { Fragment } from 'react';
+import { Example, Props } from '@kata-kit/doc-utils';
+
+import { Card, CardGrid, DashboardCard } from '../src/index';
+import { Button } from '../../button/src/index';
+import { variables } from '../../theme/src/index';
+
 This document is a stub. Please help [expand it](https://github.com/kata-ai/kata-kit).
```

Comment:
I think we can remove this

Reply:
Haha, I forgot I left this there. I just pushed additional docs for card, though, so this is still needed.